Reproduce agents (#116)
HumanCompatibleAI/overcooked_ai: src/overcooked_demo/README.md

## Use the human vs. human game mode.

### 1): Using Docker Compose CLI to deploy on EC2

> With the Overcooked demo, you can test the interaction between two human players. To do this, you need to deploy this code on the server (hsttps://docs.docker.com/language/python/deploy/). 

Reviewer: ```suggestion
With the Overcooked demo, you can test the interaction between two human players. To do this, you need to deploy this code on a server (hsttps://docs.docker.com/language/python/deploy/). 
```

"a" server? what do you mean by "the"?

Author: hmmm I didn't write this part. I think it was there on the old readme and I just copied and pasted it. I guess they are synonymous in this context but ..... I'll just change it.

After successful deployment, the first user should open http://[server_ip_address]/, select the human keyboard input for both players and click on "Create game". If everything has been successful, he will receive a message: "Waiting for game to start".
Another user should open a page at http://[server_ip_address]/psiturk to start the game.  

If you want to run a test on a local computer, you should use "localhost" instead of "server_ip_address" and open the corresponding links in different tabs.

### 2): Setup the repo on a cloud server 
One also has the option to setup the repo again on a cloud server. This allows more flexible customization and easier spin-up. The following steps demonstrate how one can steup the repo on an AWS EC2 instance 

> 1: Get an [AWS account](https://aws.amazon.com/free/)

Reviewer: Does it work with the free account? If not, let's not link to that. Also worth mentioning that it doesn't require using AWS but we only talk about AWS instructions

Author: Yeah it should work fine, and the signup portal is the same for all types of accounts. With a free account you also get access to not free instances (it is just you can't use the really good ones like P-type instances with GPUs without building a history of good credits), but for the purpose of running overcooked games a free account should be sufficient.

### 2): Setup the repo on a cloud server 
One also has the option to setup the repo again on a cloud server. This allows more flexible customization and easier spin-up. The following steps demonstrate how one can steup the repo on an AWS EC2 instance 

> 1: Get an [AWS account](https://aws.amazon.com/free/). Note any cloud server should work and it is by no means limited to AWS, and the instruction provided here is only meant to give an example for how it can be done.

Reviewer: ```suggestion
1: Get an [AWS account](https://aws.amazon.com/free/). Note that any cloud provider should work – the instructions provided here for AWS are only meant as an example.
```
